contrib/packages/deb/ubuntu-xenial/debian/rules

```diff
@@ -180,6 +166,9 @@ install: build
 	# tigervncserver
 	make install DESTDIR=$(CURDIR)/debian/tigervncserver
 	(cd unix/xserver/hw/vnc; make install DESTDIR=$(CURDIR)/debian/tigervncserver)
+	# TODO: why does viewer stuff end up in server installation?
```

Comment:
Because there is no such thing as a server installation. It looks like the rules file need some general fixing. The proper approach is to do what the RPM build does, i.e. run `make install` in the top directory and separate things after that. Anything else is likely to break.

This should probably be fixed in a separate commit for 12.04 and 14.04 before the 16.04 packaging.